interfaces/builtin/all_test.go

```diff
@@ -200,10 +200,10 @@ type snippetDefiner2 interface {
 	ConnectedSlotSnippet(plug *interfaces.Plug, slot *interfaces.Slot, sec interfaces.SecuritySystem) error
 }
 type snippetDefiner3 interface {
-	PermanentPlugSnippet(plug *interfaces.Plug, sec interfaces.SecuritySystem) error
+	PermanentPlugSnippet(plug *snap.PlugInfo, sec interfaces.SecuritySystem) error
 }
 type snippetDefiner4 interface {
-	PermanentSlotSnippet(slot *interfaces.Slot, sec interfaces.SecuritySystem) error
+	PermanentSlotSnippet(slot *snap.SlotInfo, sec interfaces.SecuritySystem) error
 }
```

Comment:
All the changes above are ok. Can you add a test that checks we are not merging any new interfaces with the old signature methods please?

Reply:
On second thought #4124 ensures this directly so please just merge it here after it lands in master.